### PR TITLE
update README to use recommended approach for reactiveServices

### PR DESCRIPTION
**Pull request by dvagala**
In the readme, there was no mention of the correct usage of the reactiveServices. I removed the old approach with ReactiveValues altogether, I've just written there a little side note. Let me know if you think the old approach should take space in the readme.

[Stacked slack conversation about this.](https://filledstacks.slack.com/archives/CJY4SV5KQ/p1664583320808749)

### CODE REVIEW

File: packages/stacked/README.md [hunk 1]
```diff
@@ -592,38 +592,23 @@ One thing that was common in a scenario with the first implementation of this ar
...
-class InformationService with ReactiveServiceMixin { //1
-  InformationService() {
-    //3
-    listenToReactiveValues([_postCount]);
-  }
-
-  //2
-  ReactiveValue<int> _postCount = ReactiveValue<int>(0);
-  int get postCount => _postCount.value;
+class PostsReactiveService with ReactiveServiceMixin {
```
Comment (FilledStacks) [inline packages/stacked/README.md]: Please remove Reactive from the name. That's not a pattern that I think we should teach

File: packages/stacked/README.md [hunk 2]
```diff
@@ -592,38 +592,23 @@ One thing that was common in a scenario with the first implementation of this ar
...
-
-There are three things you need to make a service reactive.
-
-1. Use the `ReactiveServiceMixin` with the service you want to make reactive
-2. Wrap your values in an ReactiveValue.
-3. Register your reactive values by calling `listenToReactiveValues`. A function provided by the mixin.
-
-Below is some source code for the non-theory coders out there like myself.
+In the stacked library, we have a `ReactiveServiceMixin` which can be used to store a state that is needed across multiple ViewModels. The approch is simple. Just set the state variable and call `notifyListeners()`. This will notify any ViewModel that registered this `ReactiveServiceMixin`, and UI will be updated accordingly.
```
Comment (FilledStacks) [inline packages/stacked/README.md]: approch -> approach

File: packages/stacked/README.md [hunk 3]
```diff
@@ -633,12 +618,12 @@ This ViewModel extends the `BaseViewModel` and adds a function that allows you t
 2. Implement `reactiveServices` getter that returns a list of reactive services.
 
 ```dart
-class WidgetOneViewModel extends ReactiveViewModel {
-  // You can use get_it service locator or pass it in through the constructor
-  final InformationService _informationService = locator<InformationService>();
+class AnyViewModel extends ReactiveViewModel {
+  final _postsReactiveService = locator<PostsReactiveService>();
```
Comment (FilledStacks) [inline packages/stacked/README.md]: Update these as well to remove `Reactive` from the name

**Review by FilledStacks — CHANGES_REQUESTED** [review]
Awesome! Thank you for the update. Just comments for changes. In addition to those comments you should also:

- Update the version of stacked in the pubspec to match the changelog.md entry.

**Comment by dvagala** [comment]
@FilledStacks thanks, I've corrected those issues

**Review by FilledStacks — APPROVED** [review]
Awesome!! Merging in now

**Comment by FilledStacks** [comment]
I'll have to fix this PR before I can merge it in. I'll see if I can find some time for it in the week after my releases are out.

**Review by FilledStacks — APPROVED** [review]
Gonna merge this the CI is breaking for a weird reason.

**Pull request merged**